packages/drafts-realm/ai-app-generator.gts

```diff
@@ -0,0 +1,452 @@
+import { type CardDef } from 'https://cardstack.com/base/card-api';
+import {
+  CardContainer,
+  FieldContainer,
+  BoxelInput,
+  Button,
+} from '@cardstack/boxel-ui/components';
+import { cssVar, eq, getContrastColor } from '@cardstack/boxel-ui/helpers';
+import { fn } from '@ember/helper';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import GlimmerComponent from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { AppCard, Tab } from './app-card';
+
+const getComponent = (card: CardDef) => {
+  if (!card) {
+    return;
+  }
+  return card.constructor.getComponent(card);
+};
+
+class Requirements extends GlimmerComponent<{
+  instances: CardDef[] | [];
+}> {
+  <template>
+    <div class='requirements'>
+      <aside class='recent-reqs-sidebar'>
+        <h3 class='recent-reqs-title'>Recent Requirements</h3>
+        <ul>
+          {{#each @instances as |doc|}}
+            <li>
+              <button {{on 'click' (fn this.openDoc doc)}}>
+                {{doc.title}}
+              </button>
+            </li>
+          {{/each}}
+        </ul>
+      </aside>
+      {{#let (getComponent this.currentDoc) as |Card|}}
```

Comment:
This is fine for now but I wanted to note that it is an example of the awkwardness we have with the @fields/@model dichotomy